src/documents/congratulations.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Congratulations
+layout: default
+category: API
+---
+
+## Overview
+
+**Component Name:** `gpii.firstDiscovery.panel.congratulations`
+
+**File:** `panels.js`
+
+Displays the congratulations message.
+
+## Using the Congratulations Panel
+
+*Option 1*: In the
+[Auxiliary Schema](http://docs.fluidproject.org/infusion/development/AuxiliarySchemaForPreferencesFramework.html),
+specify the name of the panel as the `type` property of the `panel` section:
+```javascript
+"congratulations": {
+    "type": "gpii.firstDiscovery.congratulations",
+    "panel": {
+        "type": "gpii.firstDiscovery.panel.congratulations",
+        "container": ".gpiic-fd-prefsEditor-panel-congratulations",
+        "template": "%templatePrefix/congratulationsTemplate.html",
+        "message": "%messagePrefix/congratulations.json"
+    }
+}
+```
+
+*Option 2*: Outside the context of the First Discovery Tool, developers may wish to create a standalone component:
+```javascript
+var myPanel = gpii.firstDiscovery.panel.congratulations(container, options);
+```
+
+## Grades
+
+This component uses the following base
+[grades](http://docs.fluidproject.org/infusion/development/ComponentGrades.html):
+
+* [`fluid.prefs.panel`](http://docs.fluidproject.org/infusion/development/Panels.html)
+
+## Options
+
+This component can be configured using the following
+[options](http://docs.fluidproject.org/infusion/development/ComponentOptionsAndDefaults.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`selectors`</td>
+        <td>Javascript object containing selectors for various fragments of the markup, including the containers for the subcomponents.</td>
+        <td></td>
+        <td>See [Selectors](#selectors) below</td>
+    </tr>
+</table>
+
+## Selectors
+
+One of the options that can be provided to Infusion components is a set of CSS-based
+selectors identifying where in the DOM different elements can be found. Components use a
+[DOM Binder](http://docs.fluidproject.org/infusion/development/DOMBinder.html) to access the
+named elements.
+
+The value for the option is itself a Javascript object containing name/value pairs:
+
+```javascript
+selectors: {
+    selector1Name: "selector 1 string",
+    selector2Name: "selector 2 string",
+      ...
+}
+```
+
+| Selector Name | Description | Default |
+|---------------|-------------|---------|
+| `message` | The container to display the welcome message. | `".gpiic-fd-welcome-instructions"` |
```

Comment:
The description should be "... to display the congratulations message." instead of "welcome message".

The default should be ".gpiic-fd-congratulations-message".